features/step_definitions/satisfaction_report_steps.rb

```diff
@@ -0,0 +1,29 @@
+Given(/^there are existing satisfaction records$/) do
+  create_list(:satisfaction, 20)
```

Comment:
We could trim this down to just `11` I guess - and still be sure the other steps are testing what they need. Would be a little quicker.